### DISCUSSION TITLE
pluging commands missing from auto-complete after plugin setup.

### DISCUSSION THREAD

**Discussion by madelaney** [hrsh7th/nvim-cmp#1657] [General]
All,

I was following the setup instruction on the README.md and after adding the lines [113..120](https://github.com/hrsh7th/nvim-cmp/blob/main/README.md?plain=1#L113-L120), nvim plugin commands (i.e `:TSUpdate`, and `:checkhealth`) no longer show up in the auto-complete window.

I added `nvim_lua` as a source thinking that would fix it (see below for updated config) but that didn't seem to have the wanted effect.

```lua
cmp.setup.cmdline(':', {
  mapping = cmp.mapping.preset.cmdline(),
  sources = cmp.config.sources({
    { name = 'nvim_lua' },
    { name = 'path' },
    { name = 'cmdline' },
  }),
})
```

Is there a way for me to get nvim plugin commands to show up again, while also adding `path` and `cmdline` to the auto complete?

- Mike D.

**Comment by fitrh** [comment]
Can you share your config? Also, you don't need `nvim_lua` source for cmdline

**Comment by madelaney** [comment]
@fitrh ,

Roger that, I'll remove `nvim_lua` as you suggested.

Also here is my config - https://gist.github.com/madelaney/b1be9a9a2bc1a58be3dc34a08a50e5f4

- Mike D.
